resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/SparkConfPropagateSuite.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.deploy.k8s.integrationtest
+
+import java.io.{BufferedWriter, File, FileWriter}
+import java.net.URL
+
+import scala.io.{BufferedSource, Source}
+
+import io.fabric8.kubernetes.api.model._
+
+import org.apache.spark.internal.config
+
+private[spark] trait SparkConfPropagateSuite { k8sSuite: KubernetesSuite =>
+  import KubernetesSuite.{k8sTestTag, SPARK_PI_MAIN_CLASS}
+
+  test("Verify logging configuration is picked from the provided SPARK_CONF_DIR/log4j.properties",
```

Comment:
Ur, is the test name correct? `log-config-test-log4j.properties` seems to be used in the test.

Reply:
That is the name to keep it distinct from existing `log4j.properties`. However, when the spark conf dir is created, the default logger file name is log4j.properties.